<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.ebay.pagesTest.FashionPageTest" tests="3" time="48.078" ignored="4" failures="1" timestamp="14 Sep 2017 16:34:05 GMT" skipped="0" hostname="Parul" errors="0">
  <testcase name="searchData" time="20.702" classname="com.ebay.pagesTest.FashionPageTest">
    <failure type="java.lang.AssertionError" message="expected:&lt;dress in Fashion | eBay&gt; but was:&lt;dress in Fashion| eBay&gt;">
      <![CDATA[java.lang.AssertionError: expected:<dress in Fashion | eBay> but was:<dress in Fashion| eBay>
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
	at com.ebay.pagesTest.FashionPageTest.searchData(FashionPageTest.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- searchData -->
  <testcase name="searchData" time="13.324" classname="com.ebay.pagesTest.FashionPageTest"/>
  <testcase name="searchData" time="14.052" classname="com.ebay.pagesTest.FashionPageTest"/>
  <testcase name="checkhm" classname="com.ebay.pagesTest.FashionPageTest">
    <skipped/>
  </testcase> <!-- checkhm -->
  <testcase name="beforeSuite" classname="com.ebay.base.BaseTest">
    <skipped/>
  </testcase> <!-- beforeSuite -->
  <testcase name="beforeSuite" classname="com.ebay.base.BaseTest">
    <skipped/>
  </testcase> <!-- beforeSuite -->
  <testcase name="toplistTest" classname="com.ebay.pagesTest.HomePageTest">
    <skipped/>
  </testcase> <!-- toplistTest -->
</testsuite> <!-- com.ebay.pagesTest.FashionPageTest -->
